x/posts/types/models.go

```diff
@@ -584,6 +664,15 @@ func IsPoll(attachment Attachment) bool {
 	return ok
 }
 
+// IsActivePoll tells whether the given attachment represents a active poll or not
+func IsActivePoll(attachment Attachment) bool {
```

Comment:
Instead of relying on a check of `FinalTallyResulsts` I think it's best to check the storage instead, running through the list of active polls. This would make sure that if there is any bug with the setting of the tally results we do not end up with a bug here as well